tests/packages/abc/prog/abc4.cpp

```diff
@@ -2,6 +2,10 @@
 
 using namespace std;
 
+int rnd(int a, int b) {
+	return a + rand() % (b - a + 1);
+}
+

```

Comment:
This function here is not needed, right?

Reply:
Yes, I deleted it: 486727169eab6172e00a446bf84c28e1173c8ee1